Documentation/gettingstarted/minikube.rst

```diff
@@ -20,6 +20,441 @@ deploying Cilium on a full fledged Kubernetes cluster, then go straight to
+
+      kubectl create -f \ |SCM_WEB|\/examples/kubernetes/1.9/cilium-minikube.yaml
+
+  .. group-tab:: K8s 1.8
+
+    .. parsed-literal::
+
+      kubectl create -f \ |SCM_WEB|\/examples/kubernetes/1.8/cilium-minikube.yaml
+
+
+Step 3: Deploy the Demo Application
+===================================
+
+Now that we have Cilium deployed and ``kube-dns`` operating correctly we can deploy our demo application.
+
+In our Star Wars-inspired example, there are three microservices applications: *deathstar*, *tiefighter*, and *xwing*. The *deathstar* runs an HTTP webservice on port 80, which is exposed as a `Kubernetes Service <https://kubernetes.io/docs/concepts/services-networking/service/>`_ to load-balance requests to *deathstar* across two pod replicas. The *deathstar* service provides landing services to the empire's spaceships so that they can request a landing port. The *tiefighter* pod represents a landing-request client service on a typical empire ship and *xwing* represents a similar service on an alliance ship. They exist so that we can test different security policies for access control to *deathstar* landing services.
+
+**Application Topology for Cilium and Kubernetes**
+
+.. image:: images/cilium_http_gsg.png
+   :scale: 30 %
+
+The file ``http-sw-app.yaml`` contains a `Kubernetes Deployment <https://kubernetes.io/docs/concepts/workloads/controllers/deployment/>`_ for each of the three services.
+Each deployment is identified using the Kubernetes labels (``org=empire, class=deathstar``), (``org=empire, class=tiefighter``),
+and (``org=alliance, class=xwing``).
+It also includes a deathstar-service, which load-balances traffic to all pods with label (``org=empire, class=deathstar``).
+
+.. parsed-literal::
+
+    $ kubectl create -f \ |SCM_WEB|\/examples/minikube/http-sw-app.yaml
+    service "deathstar" created
+    deployment "deathstar" created
+    deployment "tiefighter" created
+    deployment "xwing" created
+
+
+Kubernetes will deploy the pods and service in the background.  Running
+``kubectl get pods,svc`` will inform you about the progress of the operation.
+Each pod will go through several states until it reaches ``Running`` at which
+point the pod is ready.
+
+::
+
+    $ kubectl get pods,svc
+    NAME                             READY     STATUS    RESTARTS   AGE
+    po/deathstar-76995f4687-2mxb2    1/1       Running   0          1m
+    po/deathstar-76995f4687-xbgnl    1/1       Running   0          1m
+    po/tiefighter                    1/1       Running   0          1m
+    po/xwing                         1/1       Running   0          1m
+
+    NAME             TYPE        CLUSTER-IP       EXTERNAL-IP   PORT(S)   AGE
+    svc/deathstar    ClusterIP   10.109.254.198   <none>        80/TCP    3h
+    svc/kubernetes   ClusterIP   10.96.0.1        <none>        443/TCP   3h
+
+Each pod will be represented in Cilium as an :ref:`endpoint`. We can invoke the
+``cilium`` tool inside the Cilium pod to list them:
```

Comment:
should the `kubectl get cep` be used here?

Reply:
This is from the existing guide. I've only reworked the Cilium installation part for now.